### ISSUE TITLE
Deleted user not recognized correctly

### ISSUE THREAD

**Issue by shenef** [Avasam/speedrun.com_global_scoreboard_webapp#171]
User "Kenata8" (98rpmd6j) has been deleted on srcom but instead of showing the deleted user message it just shows the "not recognized, use ID instead" message, even when already using the ID.

**Comment by Avasam** [comment]
I think this is working as intended. SRC's API makes no mention of that user (see: https://www.speedrun.com/api/v1/users/98rpmd6j ).

If a user is missing from SRC, but present in the scoreboard's database, they are simply cleaned out. From that point onward, no one is aware of the existence of that user. Trying to update them is no more different that trying to update a user that does not exist, because well, they don't.

In other words. The deleted user message only appears once if that user had previously been in the scoreboard.

**Comment by shenef** [comment]
Ah i see, it indeed seems to work as intended. I was assuming it wasn't working since i got the same result even 10 minutes later after refreshing the page.
But now after cleaning the cache and doing a hard reload this entry seems to be gone, maybe it was caused by caching on my side?
I'll be closing this, since if there actually was a problem, it lies elsewhere.
